turbinia/turbiniactl.py

```diff
@@ -332,7 +332,7 @@ def main():
     args.name = args.name if args.name else args.local_path
     local_path = os.path.abspath(args.local_path)
     evidence_ = evidence.RawDisk(
-        name=args.name, local_path=local_path,
+        name=args.name, source_path=local_path,
```

Comment:
Which flags? In turbiniactl?
I guess it could make sense, but this is a User facing change, we'd also need to update all the docs. Maybe another PR?

Reply:
Yeah, in turbiniactl.  I'm fine with doing that in a separate PR, but I would eventually like them consistent just because there are already so many different path names that it's hard to keep track, :).

Reply:
done